hydra/controllers/client.py

```diff
@@ -279,6 +280,20 @@ def apply_jumpstart(self):
         else:
             self.app.log.warning(f'Service not installed.  You will need to restart your node manually.')
 
+
+    @ex(arguments = [
```

Comment:
Was this committed by accident? It looks like a decorator for the leave-network command, I can't figure out what it's doing here.

Reply:
Yep, copy/ pasted and forgot to remove.